set.mm

```diff
@@ -21576,7 +21576,8 @@ proposition with a distinct variable (closed form of ~ nfsb4 ).
     $( Equivalence for substitution.  (Contributed by NM, 2-Jun-1993.)  (Proof
        shortened by Wolf Lammen, 23-Sep-2018.) $)
     sb6a $p |- ( [ y / x ] ph <-> A. x ( x = y -> [ x / y ] ph ) ) $=
-      ( wsb weq wi wal sbco sb6 bitr3i ) ABCDACBDZBCDBCEKFBGABCHKBCIJ $.
+      ( wsb weq wi wal sbcom3vv sb6 sbid sbbii 3bitr3ri ) ACBDZBCDACCDZBCDBCEMF
+      BGABCDACBCHMBCINABCACJKL $.
```

Comment:
It may be more versatile to prove a "sbcov" from "sbcom3vv" in the same way that sbco is prove from sbcom3.

Then, sb6a should be moved to before ax-13.

Reply:
I committed the suggestion, is the new placement good?